telemetry/definitions/commonDefinitions.json

```diff
@@ -1457,6 +1497,31 @@
             "type": "int",
             "description": "Number of generations before the user accepted or rejected"
         },
+        {
+            "name": "numberOfBuildExecuted",
+            "type": "int",
+            "description": "Number of build executed in unit test generation build loop"
+        },
+        {
+            "name": "numberOfBuildFailed",
+            "type": "int",
+            "description": "Number of build failures in unit test generation build loop"
+        },
+        {
+            "name": "numberOfBuildPassed",
+            "type": "int",
+            "description": "Number of build succeeded in unit test generation build loop"
+        },
```

Comment:
there are already some generic "successCount" and similar fields. use those. if you need to add a `executedCount` or something like that, that is preferred.

do not add over-specific fields for every metric.